webextension/privileged/ui/CloudDownloadsView.jsm

```diff
@@ -236,6 +236,10 @@ var CloudDownloadsView = {
     const providers = await CloudStorage.getStorageProviders();
     const keys = Array.from(providers.keys());
 
+    if (!this.eventEmitter) {
```

Comment:
eventEmitter is set inside cloudStorage.js when add-on is installed or on subsequent browser restarts. If eventEmitter is null we should exit.